host/3.0/java-build.yml

```diff
@@ -75,6 +102,17 @@ steps:
     displayName: java8-appservice
     continueOnError: false
 
+  - bash: |
+      set -e
+      IMAGE_NAME=azurefunctions.azurecr.io/azure-functions/3.0/java:$(Build.SourceBranchName)-java8.1-appservice
```

Comment:
Just a callout here. The new Java images are tagged with their version.1 as seen here.

Reply:
so 8.1 and 11.1 will be the old images with zulu jdk. new image with new jdk will be as 8 and 11.